build/dev/haproxy_local_setup.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+echo "Re/Create dss/crdb cluster setup  using HAProxy"
+RESULTFILE="$(pwd)/haproxy_cluster_setup"
+touch "${RESULTFILE}"
+cat /dev/null > "${RESULTFILE}"
+FLAGS="--network dss_sandbox_default cockroachdb/cockroach:v21.2.3 start --insecure --join=roacha,roachb,roachc"
+
+
+OS=$(uname)
+if [[ "$OS" == "Darwin" ]]; then
+	# OSX uses BSD readlink
+	BASEDIR="$(dirname "$0")"
+else
+	BASEDIR=$(readlink -e "$(dirname "$0")")
+fi
+
+cd "${BASEDIR}/../.." || exit 1
+
+DC_COMMAND=$*
+
+if [[ "$DC_COMMAND" == "down" ]]; then
+  cleanup || true
```

Comment:
Trying to call `cleanup` function created below to stop the instances, but this call is not working at the moment. Let me know what is the right way to call this function. `call :cleanup` is also not working.